packages/startup/entry/sample-modules/ai-native/ai-native.contribution.ts

```
@@ -493,4 +498,38 @@ export class AINativeContribution implements AINativeCoreContribution {
       }
     });
   }
+
+  registerChatAgentPromptProvider(): void {
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)